tests/functional/centos/7/ooo-collocation/hosts

```diff
@@ -0,0 +1,92 @@
+    ceph_repository: community
+    ceph_release: luminous
+    ceph_stable: true
+    cephfs: cephfs
+    cephfs_data: manila_data
+    cephfs_metadata: manila_metadata
+    cephfs_pools:
+    - {name: manila_data, pgs: 8}
+    - {name: manila_metadata, pgs: 8}
+    cluster_network: 192.168.96.0/24
+    containerized_deployment: true
+    devices: [ '/dev/sda', '/dev/sdb' ]
+    docker: true
+    fsid: 2b42b1be-0eca-11e8-8b0e-52540057effe
+    generate_fsid: false
+    ip_version: ipv4
+    ireallymeanit: 'yes'
+    keys:
+    - {key: AQASm39aAAAAABAAMbQ1Qi4mVLvE1zlw6emUaA==, mgr_cap: allow *, mode: '0600',
+      mon_cap: allow r, name: client.openstack, osd_cap: 'allow class-read object_prefix
+        rbd_children, allow rwx pool=volumes, allow rwx pool=backups, allow rwx pool=vms,
+        allow rwx pool=images, allow rwx pool=metrics'}
+    - {key: AQASm39aAAAAABAA0DhN14/TTIy8OxGaJL25RQ==, mds_cap: allow *, mgr_cap: allow
+        *, mode: '0600', mon_cap: 'allow r, allow command \\\"auth del\\\", allow
+        command \\\"auth caps\\\", allow command \\\"auth get\\\", allow command \\\"auth
+        get-or-create\\\"', name: client.manila, osd_cap: allow rw}
+    - {key: AQASm39aAAAAABAASkRRPrzTgqOllVQKBOWrPQ==, mgr_cap: allow *, mode: '0600',
+      mon_cap: allow rw, name: client.radosgw, osd_cap: allow rwx}
+    mds_use_fqdn: false
+    mon_use_fqdn: false
+    monitor_address_block: 192.168.95.0/24
+    monitor_secret: AQASm39aAAAAABAAS9tc7s7m5VNFPdYWEF+aLA==
+    ntp_service_enabled: false
+    openstack_config: true
+    openstack_keys:
+    - {key: AQASm39aAAAAABAAMbQ1Qi4mVLvE1zlw6emUaA==, mgr_cap: allow *, mode: '0600',
+      mon_cap: allow r, name: client.openstack, osd_cap: 'allow class-read object_prefix
+        rbd_children, allow rwx pool=volumes, allow rwx pool=backups, allow rwx pool=vms,
+        allow rwx pool=images, allow rwx pool=metrics'}
+    - {key: AQASm39aAAAAABAA0DhN14/TTIy8OxGaJL25RQ==, mds_cap: allow *, mgr_cap: allow
+        *, mode: '0600', mon_cap: 'allow r, allow command \\\"auth del\\\", allow
+        command \\\"auth caps\\\", allow command \\\"auth get\\\", allow command \\\"auth
+        get-or-create\\\"', name: client.manila, osd_cap: allow rw}
+    - {key: AQASm39aAAAAABAASkRRPrzTgqOllVQKBOWrPQ==, mgr_cap: allow *, mode: '0600',
+      mon_cap: allow rw, name: client.radosgw, osd_cap: allow rwx}
+    openstack_pools:
+    - {name: images, pg_num: 8, rule_name: ''}
+    - {name: metrics, pg_num: 8, rule_name: ''}
+    - {name: backups, pg_num: 8, rule_name: ''}
+    - {name: vms, pg_num: 8, rule_name: ''}
+    - {name: volumes, pg_num: 8, rule_name: ''}
+    osd_objectstore: filestore
+    osd_scenario: collocated
+    pools: []
+    public_network: 192.168.95.0/24
+    user_config: true
+clients:
+  hosts:
+    client0: {}
+    mon0: {}
+mdss:
+  hosts:
+    mon0: {}
+mgrs:
+  hosts:
+    mon0: {}
+mons:
+  hosts:
+    mon0: {}
+nfss:
+  hosts: {}
+osds:
+  hosts:
+    mon0: {}
```

Comment:
I'm unsure if the CI will allow us to collocate an OSD with a MON. The MON nodes do not get the same device setup and private/public network as the OSD nodes do.

Reply:
That's OK, we don't need the OSD collocated with the mon.